api-docs/README.md

```diff
@@ -0,0 +1,24 @@
+# Tock API documentation
+
+## Before you start
+You'll need to have an API key, and you'll need to have it added to the `GSA_TE_Tock_Library` script. Contact a Tock developer or ping the `#tock-dev` slack channel for help.
```

Comment:
I’m not sure this makes sense here. As the API documentation is for our API, but as Tock devs, we don’t take ownership of external libraries using it. Perhaps we should? Then I’d say that the source for those libs should be in this directory under, GAS examples or something. Wdyt?

Reply:
Well, the docs were already in this folder, just lacking an index/readme. So by creating one we improve the navigability and usefulness of those docs, provide some brief summary AND have a handy spot to say, "Hey, API user, you're gonna need a key."

That was my thinking anyway.